src/math/MathUtils.js

```diff
@@ -291,7 +291,32 @@ function normalize( value, array ) {
 
 }
 
-
+const MathUtils = {
+	DEG2RAD: DEG2RAD,
+	RAD2DEG: RAD2DEG,
+	generateUUID: generateUUID,
+	clamp: clamp,
+	euclideanModulo: euclideanModulo,
+	mapLinear: mapLinear,
```

Comment:
Would it not be better use just vars instead of `prop: var` like?

```js
const MathUtils = {
	DEG2RAD,
	RAD2DEG,
	generateUUID,
	...
```

Reply:
It's exactly the same thing and gets translated to the code referenced.  This style allows the internal names to be changed (many modules import one or two directly bypassing this), while ensuring the documented API is always the same.

Reply:
Last bit, when it comes to Terser and minification, these interface objects reduce the bundle size.  When the public API is used exclusively, the function names themselves can be condensed into a 2 character symbol, and used that way internally.  This marks the only spot where that 2 character replacement needs to appear in its full form.

Reply:
I think it's cleaner than replicating the same names all times. For this specific case, I don't see the need to use it `prop:prop` instead of `prop`, or do we have some compatibility issue here?

Reply:
Minification issue, sorry, I wrote that out just as you were responding.

So if any function names change, you have to introduce this object and it will be created anyway.  I'm just trying to avoid creating foot-guns.

Reply:
> The alternative is if any function names change, you have to introduce this format to maintain the API, it will be created anyway if something is inline-d. 

But why we would change a function name? And, if we would decide that a new name is better than the old one, we will also update the external name.

`{ prop1, prop2, ... }` format is also used in other places throught the library.

Reply:
What's the goal of this feedback ?

> But why we would change a function name? 

API's and function names changing with 10 version support seems to be the norm.  So much so, there's a file dedicated to it with new updates every release, `Three.legacy.js`.  

> { prop1, prop2, ... } format is also used in other places through-out the library.

I didn't write that other code.  I hope the tradeoffs and reasons were well considered.  Plenty of places where things are not as frequently imported / exported where it does not matter because it's not API.

Reply:
> What's the goal of this feedback ?

Because the code style is such in files like this, for example, BufferGeometryUtils. I think we can discuss it in another issue if it should be changed -- I suggest that you use the `{ prop1, prop2, ...` format for now.

Reply:
See the reasoning above.

Reply:
I still don't agree. But I think @mrdoob and @Mugen87 should decide.

Reply:
Yes, the final minimized size.  

The idea of removing minified builds is crazy.  If you are importing a library into your website from a CDN, you're not shipping un-minified dependencies in production.  Seems kinda odd for people to use a CDN to improve load times but, then lose that time because the build does not have a minified distribution.

Maybe there should be a document or readme about why things are done.